### DISCUSSION TITLE
surface runoff in the fully coupled WRF/WRF-Hydro model

### DISCUSSION THREAD

**Discussion by Ordosnanjingdyf** [NCAR/wrf_hydro_nwm_public#750] [Q&A] [answered]
Hello，WRF-Hydro Team：
When running the fully coupled WRF/WRF-Hydro model, I encountered an issue with the surface runoff output in the model. I found that only by disabling OVRTSWCRT, which is turning off overland flow, could I obtain the correct surface runoff output in the wrfout file.
I want to use the fully coupled WRF/WRF-Hydro model to calculate surface runoff. How can I obtain the correct output variables for surface runoff?

**Comment by aubreyd** [comment, marked as answer]
Yes, Noah-MP's SFCRNOFF output variable is not valid when you have WRF-Hydro's overland routing active. This is because WRF-Hydro will do explicit routing of overland flow, so you will not use Noah-MP's more conceptual representation of surface runoff. When you are using WRF-Hydro's overland routing, you should look at WRF-Hydro's sfcheadsubrt variable (in RTOUT) and/or qSfcLatRunoff (in CHRTOUT).

**Reply by Ordosnanjingdyf** [reply]
Thank you very much for your response. Apologize for the inconvenience, I have another question to consult with you:
My WRF version is 4.1.2, and the WRF-Hydro version is 5.1.1. I have checked the variable "sfcheadsubrt" in the output file RTOUT in the fully coupled mode. My understanding is that "sfcheadsubrt" represents instantaneous surface water accumulation (in mm/DT, where DT is the integration time step of WRF-Hydro).
However, it seems that the variable "sfcheadsubrt" is actually 1000 times the real value. I suspect that there may be a missing scale factor (0.001) here. Is this discrepancy caused by conversion between different units (e.g., m and mm)?
Looking forward to your reply.

Best regards,
Yifan Dong

**Reply by aubreyd** [reply]
sfcheadsubrt is not a flux, it is a state (ponded water depth).I believe the units are in mm but you should be able to check the output variable metadata for the units.

**Comment by Ordosnanjingdyf** [comment]
Thank you very much for your reply, and&nbsp;I apologize for bothering you once again.&nbsp;
I will review the units of the variables.
Best regards!
&nbsp;


凡蛋儿
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Aubrey ***@***.***&gt;; 
发送时间: 2024年5月23日(星期四) 凌晨3:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [NCAR/wrf_hydro_nwm_public] surface runoff in the fully coupled WRF/WRF-Hydro model (Discussion #750)














 
sfcheadsubrt is not a flux, it is a state (ponded water depth).I believe the units are in mm but you should be able to check the output variable metadata for the units.
